airbyte-integrations/connectors/source-facebook-marketing/integration_tests/spec.json

```diff
@@ -40,16 +42,92 @@
         "description": "The date until which you'd like to replicate data for all incremental streams, in the format YYYY-MM-DDT00:00:00Z. All data generated between the start date and this end date will be replicated. Not setting this option will result in always syncing the latest data.",
         "order": 3,
         "pattern": "^$|^[0-9]{4}-[0-9]{2}-[0-9]{2}T[0-9]{2}:[0-9]{2}:[0-9]{2}Z$",
-        "examples": ["2017-01-26T00:00:00Z"],
+        "examples": [
+          "2017-01-26T00:00:00Z"
+        ],
         "type": "string",
         "format": "date-time"
       },
-      "include_deleted": {
-        "title": "Include Deleted Campaigns, Ads, and AdSets",
-        "description": "Set to active if you want to include data from deleted Campaigns, Ads, and AdSets.",
-        "default": false,
+      "campaign_statuses": {
+        "title": "Campaign Statuses",
+        "description": "Select statuses you want to to be loaded in the stream. Empty means all available.",
```

Comment:
Why here is empty, but not all tag used to use all available?

Reply:
Thank you. Forgot to update test spec. Fixed.